Shell/NewCNF.cpp

```diff
@@ -39,7 +39,7 @@ using namespace Kernel;
 
 namespace Shell {
 
-void NewCNF::clausify(FormulaUnit* unit,Stack<Clause*>& output)
+void NewCNF::clausify(FormulaUnit* unit,Stack<Clause*>& output, DHMap<unsigned, Term*>* bindings)
```

Comment:
@quickbeam123 - I think you're the expert here. Any objections?

Reply:
Yes, look reasonable!